src/common.cpp

```diff
@@ -1689,7 +1689,7 @@ bool is_forked_child(void) {
 
     bool is_child_of_fork = (getpid() != initial_pid);
     if (is_child_of_fork) {
-        printf("Uh-oh: %d\n", getpid());
+        wprintf(L"Uh-oh: %d\n", getpid());
```

Comment:
I agree, but that should include a decision about which ones should be converted to stderr, and I don't want to do that for 2.5.0, since the beta is set for tomorrow.